pgscatalog.core/src/pgscatalog/core/lib/_normalise.py

```diff
@@ -235,49 +170,17 @@ def remap_harmonised(variants, harmonised, target_build):
             yield variant
 
 
-def check_bad_variant(variants, drop_missing=False):
-    """
-    Missing effect allele:
-
-    >>> variant = ScoreVariant(**{"effect_allele": None, "effect_weight": 5, "accession": "test", "row_nr": 0})
-    >>> list(check_bad_variant([variant], drop_missing=True)) # doctest: +ELLIPSIS
-    []
-
-    Missing chromosome name and position:
-
-    >>> variant = ScoreVariant(**{"effect_allele": "A", "effect_weight": 5, "accession": "test", "row_nr": 0})
-    >>> list(check_bad_variant([variant], drop_missing=True)) # doctest: +ELLIPSIS
-    []
-    """
-    n_bad = 0
-    for variant in variants:
-        match variant:
-            case (
-                ScoreVariant(chr_name=None)
-                | ScoreVariant(chr_position=None)
-                | ScoreVariant(effect_allele=None)
-            ):
-                # (effect weight checked separately)
-                n_bad += 1
-                if not drop_missing:
-                    yield variant
-            case _:
-                yield variant
-
-    if n_bad > 1:
-        logger.warning(f"{n_bad} bad variants")
-
-
```

Comment:
This was deleted because they're mandatory fields in the pydantic model, but this will cause an immediate crash